sdcm/tester.py

```diff
@@ -830,8 +830,8 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
             db_info['type'] = 'custom-{}-{}-ext'.format(cpu, int(mem) * 1024)
         if db_info['type'] is None:
             db_info['type'] = self.params.get('gce_instance_type_db')
-        if db_info['disk_type'] is None:
-            db_info['disk_type'] = self.params.get('gce_root_disk_type_db')
+        if db_info['root_disk_type'] is None:
+            db_info['root_disk_type'] = self.params.get('gce_root_disk_type_db')
```

Comment:
We can hardcode this to `pd-ssd` - search your e-mail box for "GCE root disk types"

Reply:
Better to hardcode it in defaults or in the `DiskStuctProvder` dataclass, I think. We might change our mind in the future and also someone joining the team might expect default values to not be buried somewhere in code.